lib/fined.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs');
+const path = require('path');
+
+const isString = require('lodash.isstring');
+const isArray = require('lodash.isarray');
+const isPlainObject = require('lodash.isplainobject');
+const isEmpty = require('lodash.isempty');
+const pick = require('lodash.pick');
+const defaults = require('lodash.defaults');
+
+const expandTilde = require('expand-tilde');
+const parsePath = require('parse-filepath');
+
+
+function fined(pathObj, defaultObj) {
+  var expandedArr = expandPaths(pathObj, defaultObj);
+  for (var i = 0, n = expandedArr.length; i < n; i++) {
```

Comment:
@phated Yes. This part was a function `firstUnNullElem` but I think it may be misunderstandable and the function would be slightly slower than `for` loop block. Is it better to use a function here?